pynest/examples/Potjans_2014/README.rst

```diff
@@ -1,61 +1,108 @@
-Microcircuit Example
-=====================
+PyNEST Microcircuit
+===================
 
-Description
------------
+This is a PyNEST implementation of the microcircuit model by Potjans and Diesmann [1]_.
+The network model represents four layers of cortex, L2/3, L4, L5, and L6, each consisting of two populations of excitatory and inhibitory neurons.
 
- This is a PyNEST implementation of the microcircuit model by Potjans and
- Diesmann [1]_ The original sli version can be found `here <https://github.com/nest/nest-simulator/tree/master/examples/nest/Potjans_2014>`__.
+.. |img1| image:: microcircuit.png
 
--  This example contains several files:
+.. |img2| image:: reference_data/raster_plot.png
 
-   -  ``helpers.py``
-      Helper functions for the simulation and evaluation of the
-      microcircuit.
-   -  ``network.py``
-      Gathers all parameters and connects the different nodes with each
-      other.
-   -  ``network_params.py``
-      Contains the parameters for the network.
-   -  ``sim_params.py``
-      Contains the simulation parameters.
-   -  ``stimulus_params.py``
-      Contains the parameters for the stimuli.
-   -  ``example.py``
-      Use this script to try out the microcircuit.
+.. |img3| image:: reference_data/box_plot.png
```

Comment:
The firing rate should be in spikes/s, not Hz.

Reply:
I changed it in the source code and it will appear in the plot when I upload a new version.